deployments/cli/community/restore-airgapped.sh

```
@@ -1,23 +1,49 @@
 }
 
+# Replace $2 (dest) with $1 (src), keeping the old dest recoverable until the
+# swap succeeds. Rolls the old data back if the move fails.
+function replaceDir() {
+    local src="$1" dest="$2" label="$3"
+    local old="${dest}.old.$$"
+
+    if [ -d "$dest" ]; then
+        mv "$dest" "$old"
+        # Restore the old data if we are interrupted mid-swap
+        trap "mv \"$old\" \"$dest\" 2>/dev/null || true; exit 1" INT TERM HUP
```

Shellcheck (0.11.0)

[warning] Use single quotes, otherwise this expands now rather than when signalled.

(SC2064)